tests/src/test/kotlin/test/OngoingStubbingTest.kt

```diff
@@ -201,6 +201,20 @@ class OngoingStubbingTest : TestBase() {
         expect(result).toBe("argument-result")
     }
 
+    @Test
+    fun testOngoingStubbing_doAnswer_withDestructuredArgument() {
```

Comment:
Nit: can you also add a test with 2 arguments?

Reply:
Sure!